tools/nl/embeddings/utils.py

```diff
@@ -156,7 +158,9 @@ def download_model_from_gcs(ctx: Context, model_folder_name: str) -> str:
       model = SentenceTransformer(downloaded_model_path)
   ```
   """
-  local_dir = ctx.tmp + "/"
+  local_dir = ctx.tmp
+  if local_dir[-1] != "/":
+    local_dir += "/"
```

Comment:
does os.path.join(local_dir, model_folder_name) not deal with it?

Reply:
It does :) Updated